azure_real_integration_test.go

```diff
@@ -14,7 +14,7 @@ import (
 //go:embed config.local.yaml
```

Comment:
syntax in this file and in app_integration_test.go is broken after signature changes.

If you use idea/goland you can add `integration` in custom tags field in go > build tags setting to see such issues more easily.